### DISCUSSION TITLE
How do I show a line graph of the number of users over time?

### DISCUSSION THREAD

**Discussion by charlesniunocturnal** [Budibase/budibase#11021] [Help]
Hi there, I can't figure out how to do this. It seems like a pretty basic use case.

I have an events table, and a users table. Each user listed in the users table has an event ID associated with them, so that you can easily figure out which event they're participating in.

I want to show a line chart, depicting a single event, with the y axis representing the number of users in the event, and the x axis representing time.

I'm very nooby with Budibase and I have no idea where to start with this. I can't even figure out how to get the total number of users in the user table, let alone users associated with a specific event, or over time. Any pointers?

**Comment by ConorWebb96** [comment]
Hey @charlesniunocturnal,

What data source are you using? E.g Budibase internal table or postgres, mysql etc?

Thanks ahead of time for this information.
